route/router.go

```diff
@@ -39,6 +40,41 @@ type Router interface {
 	Plan() store.Variable[Plan]
 }
 
+// PartialPlan is an (incomplete) Plan that operates on the internal
+// solver data structures. Certain router options that customize solver
+// internals have to work with this data structure.
+// It gives access to the following information:
+//  - Unplanned(): an Integer Domain with not yet assigned or unassigned
+//	  stops indexes.
+//  - Unassigned(): an Integer Domain with unassigned stop indexes.
+//	  These are stops explicitly excluded from being served by a vehicle.
+//  - Vehicles(): a slice of vehicles part of this partial plan.
+//  - Value(): the value of this plan.
+type PartialPlan interface {
+	Unplanned() model.Domain
+	Unassigned() model.Domain
+	Vehicles() []PartialVehicle
```

Comment:
```suggestion
	// A slice of vehicles part of this partial plan.
	Vehicles() []PartialVehicle
```
I'm going to leave this comment on this function but it should be applied to all functions. The description for the function should be placed as a normal godoc, so that intellisense will show you the documentation when attempting to summon the function. Otherwise, when you hover over the function, no documentation is shown. In general, interface methods are documented on the method definition itself. After moving the documentation of this method here, we can delete it from the general `PartialPlan` godoc.